spec/controllers/home_controller_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe HomeController, type: :controller do
+  describe "GET start page" do
+    it "has a 200 status code" do
+      get :index
+      expect(response.status).to eq(200)
+    end
+
+    it "renders index template" do
+      get :index
```

Comment:
Use DRY principle, define a [`subject`](http://www.relishapp.com/rspec/rspec-rails/v/3-2/docs/matchers/render-template-matcher#render-template-with-three-possible-options)
and use [`is_expected`](https://www.relishapp.com/rspec/rspec-core/docs/subject/one-liner-syntax) instead of `expect(subject)`